src/libstd/thread/local.rs

```diff
@@ -207,6 +209,8 @@ impl fmt::Display for AccessError {
     }
 }
 
+impl Error for AccessError {}
```

Comment:
We'll probably want a `#[stable...` attribute on this even though it does nothing today :)